.github/workflows/test-workflow.yml

```diff
@@ -1,38 +1,36 @@
-name: Tests CI
+name: Reusable test workflow
 
 on:
-  push:
-    branches: [main, develop]
-  pull_request:
-  workflow_dispatch:
+  workflow_call:
+    inputs:
+      testfilter:
+        required: true
+        type: string
 
 env:
-  RELAY_IMAGE_TAG: 0.8.0
-  NETWORK_NODE_IMAGE_TAG: 0.30.0-alpha.2
-  HAVEGED_IMAGE_TAG: 0.30.0-alpha.2
-  MIRROR_IMAGE_TAG: 0.64.0
+  RELAY_IMAGE_TAG: main
```

Comment:
Good call, maybe for the next few sprints this will be good to ensure we get everything in check.
Eventually we should switch back to a stable tag though